compiler/erg_common/error.rs

```diff
@@ -45,6 +45,7 @@ pub enum ErrorKind {
     VisibilityError = 20,
     MethodError = 21,
     DummyError = 22,
+    ExpectNewLine = 23,
```

Comment:
I've tentatively added the error numbers.
Please comment if the right number and location is available.

Reply:
👍